app/models/task.rb

```diff
@@ -0,0 +1,16 @@
+class Task < ApplicationRecord
+  enum status: { pending: 1, in_progress: 2, completed: 3 }
+  belongs_to :user, foreign_key: 'assignee_user_id'
```

Comment:
Leave one line after a group of related items (items here are enums, belongs_to assocs, has_many assocs, default validations, custom validations).